src/v2/openapi.yaml

```diff
@@ -0,0 +1,656 @@
+                - Functional
+                - Proof of Concept
+                - Unproven
+                - Not Defined
+              example: High
+            description:
+              type: string
+              example: |
+                "## Overview\r\n[`django`](https://pypi.python.org/pypi/django) is a high-level Python Web framework that encourages rapid development and clean, pragmatic design.\r\n\r\nAffected versions of this package expose sensitive information due to not properly restricting the use of a query string that performs certain object filtering. An attacker may obtain sensitive information via a series of requests containing regular expressions, as demonstrated by a `created_by__password__regex` parameter.\r\n\r\n## References\r\n- [Django Vulnerability Description](http://www.djangoproject.com/weblog/2010/dec/22/security/)\r\n- [GitHub Commit](https://github.com/django/django/commit/732198ed5c)\r\n- [Redhat Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=665373)\r\n- [Openwall](http://www.openwall.com/lists/oss-security/2011/01/03/5)\r\n- [CVE](https://web.nvd.nist.gov/view/vuln/detail?vulnId=CVE-2010-4534)\r\n"
+            fixed_in:
+              type: array
+              items:
+                type: string
+              example:
+                - 2.6.7
+                - 2.7.5
+                - 2.8.0
+            references:
+              type: array
+              items:
+                type: object
+                properties:
+                  title:
+                    type: string
+                    example: Redhat Security Advisory
+                  url:
+                    type: string
+                    example: https://access.redhat.com/security/cve/CVE-2013-2186
+          description: |
+            Extended Vulnerability details, available only to registered users
+
+    Package:
+      title: Package
+      type: object
+      required: 
+        - name
+        - version
+      properties:
+        name:
+          type: string
+          example: django
+        version:
+          type: string
+          example: 1.2.1
+        dependencies:
+          type: array
+          items:
+            $ref: '#/components/schemas/Package'
+          example:
+            - requests:
+                name: requests
+                version: 1.2.3
+            - six:
+                name: six
+                version: 5.1.1          
+      description: Package details
+
+    UnknownLicenses:
+      title: UnknownLicenses
+      type: object
+      required: 
+        - component_conflict
+        - really_unknown
+      properties:
+        component_conflict:
+          type: array
+          items:
+            type: object
+            properties:
+              license1:
+                type: string
+                example: BSD License (BSD)
+              license2:
+                type: string
+                example: General Public License V3(GPL)
+        really_unknown:
```

Comment:
can this field be renamed? It is difficult to understand what it means

Reply:
We infer to "Unknown license" in two situations,
1. really the given licenses are unknown
2. given licenses are conflicting

really_unknowns means to the first situation. I agree, it leaks the implementation detail over API. Will think of a way to abstract this.

Reply:
@yzainee Any thoughts on this?

Reply:
@arajkumar by unknown licenses, we actually mean that a particular license found in the user's stack is not available with us.

Reply:
conflicting licenses are added to "conflict_licenses"